packages/frontend/src/App.tsx

```diff
@@ -53,8 +53,8 @@ const InnerApp = () => {
     }, [error, setError]);
 
     useEffect(() => {
-        rudderAnalytics.page(undefined, 'Home');
-    }, []);
+        rudder.page(undefined, 'explore');
+    }, [rudder]);
```

Comment:
FYI: I'm refactoring this file in my WIP branch. All the page components and tracking will be in the `pages` folder.

Reply:
anything I need to do here?

Reply:
nop, was just a note for future PRs